middleware/middleware_echo.go

```diff
@@ -0,0 +1,47 @@
+package middleware
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/labstack/echo"
+)
+
+const metricName = "response_time_milliseconds"
+
+// ResponseTimeMetricsMiddleware xalala
+type ResponseTimeMetricsMiddleware struct {
+	DDStatsD *DogStatsD
+}
+
+//ResponseTimeMetricsMiddleware is a middleware to measure the response time
+//of a route and send it do StatsD
+func (responseTimeMiddleware ResponseTimeMetricsMiddleware) Serve(next echo.HandlerFunc) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		startTime := time.Now()
+		result := next(c)
+		status := c.Response().Status()
+		route := c.Path()
+		method := c.Request().Method()
+
+		timeUsed := time.Since(startTime)
```

Comment:
`timeUsed` will be in nanoseconds, correct? To keep the standards used in our node projects I think we'd better report it in milliseconds.

Reply:
My bad, we checked the dd client and we actually send a duration and it takes care of converting it to ms.